concrete/src/Block/BlockController.php

```diff
@@ -517,6 +517,14 @@ public function getPassThruActionAndParameters($parameters)
         return [$method, $parameters];
     }
 
+    /**
+     * Creates a URL that can be posted or navigated to that, when done so, will automatically run the corresponding method inside the block's controller.
```

Comment:
😍

Reply:
I just kept that fantastic phpdoc we had in BlockView::action() method ;)